com.unity.shadergraph/Documentation~/Sample-Cubemap-Node.md

```diff
@@ -2,15 +2,14 @@
 
 ## Description
 
-Samples a **Cubemap** and returns a **Vector 4** color value for use in the shader. Requires **View Direction** and **Normal** inputs to sample the **Cubemap**. You can achieve a blurring effect by sampling at a different Level of Detail using the **LOD** input. You can also define a custom **Sampler State** using the **Sampler** input.
+Samples a **Cubemap** and returns a **Vector 4** color value for use in the shader. Requires **Direction** inputs in world space to sample the **Cubemap**. You can achieve a blurring effect by sampling at a different Level of Detail using the **LOD** input. You can also define a custom **Sampler State** using the **Sampler** input.
```

Comment:
"inputs" should be "input" since there's only one

Reply:
Thanks for pointing this out! I'll fix it.